src/index.js

```diff
@@ -6,39 +7,18 @@ import { OverridableContext } from 'react-overridable';
 import { Router } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
 import { InvenioILSApp } from './lib';
+import { searchConfig } from './lib/config/searchConfig';
 
-// const CustomHome = ({ ...props }) => {
```

Comment:
nit: @ntarocco suggested in a previous review, and I would agree, it was a good idea to have them there as an example.

Reply:
we have it overwritten now in cds-books anyway, and react-overridable has examples in the documentation so IMO we can clean it up now